scalyr_agent/util.py

```diff
@@ -2203,13 +2204,14 @@ def get_agent_start_up_message():
         date_parsing_library = "native python"
 
     msg = (
-        "Starting scalyr agent... (version=%s) (revision=%s) %s (Python version: %s) "
+        "Starting scalyr agent... (version=%s) (revision=%s) %s (hostname: %s) (Python version: %s) "
```

Comment:
Since we are already changing that, I will also go ahead and change the format to foo=bar instead of foo: bar since the latter is not handled by our agent parser (that was an oversight a while back).